accounts/static/accounts/js/onboarding_base.js

```diff
@@ -0,0 +1,34 @@
+const exit_modal = document.getElementById("confirm_exit_modal");
+const overlay = document.getElementById("exit_overlay");
+console.log("HELLO");
+function redirectToLandingPage() {
+  window.location.href = "/";
+}
+
+function proceedToOnboardingGenreForm() {
+  window.location.href = "genres";
+}
+
+function closeExitModal() {
+  exit_modal.classList.add("hidden");
+  overlay.classList.add("hidden");
+}
+
+function openExitModal() {
+  exit_modal.classList.remove("hidden");
+  overlay.classList.remove("hidden");
+}
+
+/*Function from landing_page/js/index.js */
+async function getCSRFToken() {
```

Comment:
really not required, but i see this pasted around so much that it probably should be its own file that we can just grab wherever needed